### DISCUSSION TITLE
Can i use 64 bit termux with 32 bit mode?

### DISCUSSION THREAD

**Discussion by erfty125** [termux/termux-app#3032] [Q&A] [answered]
Can i use 64 bit termux with 32 bit mode?(i have 64 bit core but 32 bit mode)

**Comment by ravener** [comment, marked as answer]
No, if your phone is 64-bits but with 32-bit userspace, termux cannot use system 64-bit libraries which are missing in a 32-bit userspace.

You can still for example run 64-bit binaries that are statically linked (i.e does not link to a 64-bit system library)

**Comment by erfty125** [comment]
- Oh , ok

**Reply by Anonimatodecideavito** [reply]
Yes

**Comment by pressfanksss** [comment]
no
